src/sonic-config-engine/minigraph.py

```diff
@@ -1192,6 +1252,25 @@ def parse_xml(filename, platform=None, port_config_file=None, asic_name=None, hw
         current_device['sub_role'] = sub_role
         results['DEVICE_METADATA']['localhost']['sub_role'] =  sub_role
         results['DEVICE_METADATA']['localhost']['asic_name'] =  asic_name
+    # on Voq system each asic has a switch_id
```

Comment:
VoQ system needs "asic_id" also. The format is either simple sequential number like 0,1,2, etc., identifying the asic or PCI device id format 00:00.0. (Refer multi-asic HLD https://github.com/Azure/SONiC/blob/master/doc/multi_asic/SONiC_multi_asic_hld.md)